lib/flexmock/expectation.rb

```diff
@@ -211,6 +211,13 @@ def with_any_positional_args
       self
     end
 
+    # Declare that the method should be called with the given
+    # keyword arguments
+    def with_kw_args(kw)
```

Comment:
Wouldn't it make more sense to write this `with_kw_args(**kw)` ?

Would be weird to have to write `with_kw_args({ kw: 10 })`

Reply:
Keywords get converted to hash implicitly on single argument methods (this behavior has not changed on ruby3), plus, receiving as a positional argument allows matching against the special matches (AnyHash or the partial matcher...)